Makefile.inc1

```diff
@@ -3497,6 +3497,23 @@ delete-old-files: .PHONY
 			rm ${RM_I} $${catpage} <&3; \
 	        fi; \
 	done
+.if ${MK_MANCOMPRESS} != "yes"
+	@exec 3<&0; \
+	find ${DESTDIR}/usr/share/man ${DESTDIR}/usr/share/openssl/man ! -type d ! -name \*.gz 2>/dev/null | \
+	while read manpage; do \
+		if [ -e "$${manpage}.gz" ]; then \
+			rm ${RM_I} $${manpage}.gz <&3; \
+		fi; \
+	done
+.else
+	@exec 3<&0; \
```

Comment:
I think comments here would be good.  For this clause I think it would be something like:
```
# Remove uncompressed copies of compressed manpages.
```

Reply:
Fixed.